src/App.vue

```diff
@@ -312,20 +308,20 @@ export default class App extends Mixins(
   get breadcrumbs (): Array<BreadcrumbIF> {
     const breadcrumbs = this.$route?.meta?.breadcrumb
     const crumbs: Array<BreadcrumbIF> = [
-      getDashboardBreadcrumb(this.getEntityName, this.getBusinessDashUrl, this.getIdentifier),
+      this.getDashboardBreadcrumb(),
```

Comment:
Since this function is now in a mixin, it has full access to the store and this simplifies things.